src/CSnakes.Runtime/CSnakes.Runtime.csproj

```diff
@@ -6,6 +6,7 @@
 
     <ItemGroup>
         <PackageReference Include="pythonnet" Version="3.1.0-preview2024-07-03" />
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All"/>

```

Comment:
What does this do?

Reply:
https://learn.microsoft.com/dotnet/standard/library-guidance/sourcelink

Makes debugging into external packages easier.